internal/chart/helm_client.go

```diff
@@ -51,9 +53,11 @@ func getActionConfig(namespace string) (*action.Configuration, error) {
 	return actionConfig, nil
 }
 
-// Option to perform additional configuration of action.Install before running a chart installation.
+// InstallOption to perform additional configuration of action.Install before running a chart installation.
 type InstallOption func(install *action.Install)
 
+var _ postrender.PostRenderer = &postRender{}
```

Comment:
can we either remove it or move to `post_render.go`?

Reply:
Moved.